pkg/apis/operator/v1/logstorage_types.go

```diff
@@ -60,6 +60,12 @@ type LogStorageSpec struct {
 	// Default: tigera-elasticsearch
 	// +optional
 	StorageClassName string `json:"storageClassName,omitempty"`
+
+	// NodeSelector gives you more control over the node that Elasticsearch will run on. The contents of NodeSelector will
+	// be added to the PodSpec of the Elasticsearch nodes. For the pod to be eligible to run on a node, the node must have
+	// each of the indicated key-value pairs as labels.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
This will only apply to the ES 'worker' pods. Is there something we could add to the name to indicate that? Or should this apply to the operator as well?

Reply:
Yeah good question... should the ECK operator be goverened by the `controlPlanNodeSelector` on the Installation resource, really?

Reply:
How about `DataNodeSelector` to remove the ambiguity?